easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1032,6 +1036,69 @@ def handle_external_module_metadata(self, dep_name):
 
         return dependency
 
+    def handle_multi_deps(self):
+        """
+        Handle lists of dependency versions of which we should iterate specified in 'multi_deps' easyconfig parameter.
+
+        This is basically just syntactic sugar to prevent having to specify a list of lists in 'builddependencies'.
+        """
+
+        multi_deps = self['multi_deps']
+        if multi_deps:
+
+            # first, make sure all lists have same length, otherwise we're dealing with invalid input...
+            multi_dep_cnts = nub([len(vs) for vs in multi_deps.values()])
+            if len(multi_dep_cnts) == 1:
+                multi_dep_cnt = multi_dep_cnts[0]
+            else:
+                raise EasyBuildError("Lists of dependency versions specified in multi_deps must be of same length!")
```

Comment:
I find "Not all the dependencies in multi_deps have the same number of versions" a bit more clear. Nitpicking, I know.

Reply:
Makes sense, thanks for the suggestion